src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/SearchPageTelemetryEvent.cs

```diff
@@ -18,12 +21,16 @@ public class SearchPageTelemetryEvent : TelemetryEvent
             int pageIndex,
             int resultCount,
             TimeSpan duration,
+            IEnumerable<TimeSpan> sourceTimings,
+            TimeSpan aggregationTime,
             LoadingStatus loadingStatus) : base("SearchPage")
         {
             base["ParentId"] = parentId.ToString();
             base["PageIndex"] = pageIndex;
             base["ResultCount"] = resultCount;
             base["Duration"] = duration.TotalSeconds;
+            base["SourceTimingDuration"] = JsonConvert.SerializeObject(sourceTimings.Select(e => e.TotalSeconds));
+            base["AggregationDuration"] = aggregationTime.TotalSeconds;
```

Comment:
There are more clear alternatives to `AggregationDuration`:  `TotalDuration`, `ScenarioDuration`, `AllSourcesDuration`, ...  It's not super clear what "aggregation" means here; sum, max, min, average, etc. are all aggregations.

Reply:
AggregationDuration is the duration of the aggregation of the results from the sources. 

I get why it's confusing. 

How about `ResultsAggregationDuration` ? @dtivel

Reply:
OK, that's better.